response_operations_ui/templates/conversation-view.html

```diff
@@ -0,0 +1,43 @@
+{% extends "layouts/base.html" %}
+
+{% set subject = messages[-1]["subject"] or "No Subject" %}
```

Comment:
Is there any chance that messages could be None?  If it somehow ends up not being a list then I think this is going to break in a fairly big way and we won't be able to handle errors as we're at the template